app/models/event.py

```diff
@@ -20,6 +20,7 @@ class Event(baseModel):
     recurringId = IntegerField(null=True)
     contactEmail = CharField(null=True)
     contactName = CharField(null=True)
+    program = ForeignKeyField(Program, null=True)
```

Comment:
Are you adding this just to do Event.program? If so, why are you doing it this way instead of using ProgramEvent like we do everywhere else?

Reply:
This is part of a refactor that Fleur and Luis are working on to get rid of the many-to-many relationship in the programEvent table for a many-to-one relationship using a Program column in the Event table. Issue #792 was made to keep track of the refactoring's progress. This should make it easier to figure out what event belongs to what program and "Celts-Sponsored Events" would work as the program events that do not belong to other programs belong to.